android-kotlin/app/src/main/java/io/metropolislab/samplearchitecture/architecture/Coordinator.kt

```diff
@@ -110,7 +110,8 @@ abstract class Coordinator : ViewModelProvider.Factory {
         return true
     }
 
-    fun onActivityStarted(activity: Activity) {
+    internal fun onActivityStarted(activity: Activity) {
+        //QUESTION Why activity is not used?
```

Comment:
because i thought that we just need the first activity started no matter which as it will be the one that the coordinator launches but after thinking you're very right to ask. In the case of a login coordinator this will be lying because this won't be the main activity of the coordinator. So i guess we should check here the activity class matches the first one of the vmBindings